src/presentation/http/router/auth.test.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, test, expect } from 'vitest'
+describe('Note API', () => {
```

Comment:
we are adding tests here for the [auth router](https://github.com/codex-team/notes.api/blob/main/src/presentation/http/router/auth.ts#L35), so, it should be `Auth API` instead 
```suggestion
describe('Auth API', () => {
```